### ISSUE TITLE
Add `--force` requirement to `del --recursive`

### ISSUE THREAD

**Issue by woodb** [bloomreach/s4cmd#7]
Following s3cmd, s4cmd should consider requiring a `--force` flag for `del --recursive` calls. Because this is currently not required, unintended data loss could occur. I can submit a PR if this is in line with the project maintainer.

**Comment by chouhanyang** [comment]
Good point. I actually don't know the difference, however, this kind of change may break people's production code, so need to double check.
